app/boards/shields/kyria/kyria.overlay

```diff
@@ -28,6 +33,34 @@
 		};
 	};
 
+	default_transform: keymap_transform_0 {
```

Comment:
Moved these transforms *out* of the keymap, and into the device itself.

In the future, we'd have "variants" for keyboards for breakaway versions, e.g. `kyria_v1_five_column.overlay` that would have the alternate transform.